modules/apps/object/object-test/src/testFunctional/tests/Object.testcase

```diff
@@ -21,6 +21,143 @@ definition {
 		}
 	}
 
+	@description = "LPS-TEST - Verify if it is possible to create a Custom Object"
+	@priority = 5
+	test CanCreateCustomObject {
+		ObjectAdmin.addObjectViaAPI(
+			labelName = "Custom Object 01",
+			objectName = "CustomObject01",
+			pluralLabelName = "Custom Objects 01");
+
+		ObjectAdmin.openObjectAdmin();
+
+		Refresh();
+
+		Pause(locator1 = 3000);
+
+		AssertElementPresent(
+			key_label = "Custom Object 01",
+			locator1 = "ObjectAdmin#SELECT_OBJECT_FIELD");
+	}
+
+	@description = "LPS-TEST - Verify it is possible to create a String field"
+	@priority = 5
+	test CanCreateStringFieldInCustomObject {
+		ObjectAdmin.addObjectViaAPI(
+			labelName = "Custom Object 01",
+			objectName = "CustomObject01",
+			pluralLabelName = "Custom Objects 01");
+
+		ObjectAdmin.openObjectAdmin();
+
+		ObjectPortlet.selectCustomObject(label = "Custom Object 01");
+
+		ObjectAdmin.goToFieldsTab();
+
+		LexiconEntry.gotoAdd();
+
+		Pause(locator1 = 2000);
+
+		Type(
+			locator1 = "ObjectField#LABEL_INPUT",
+			value1 = "Custom String Field 01");
+
+		ObjectField.selectType(fieldType = "Text");
+
+		Pause(locator1 = 2000);
+
+		PortletEntry.save();
```

Comment:
These steps are not necessary because the macro I mentioned already does this. If you don't know how to use it, I recommend researching it to see how other people have used it.